src/common/options.cc

```diff
@@ -8239,6 +8239,11 @@ std::vector<Option> get_mds_options() {
      .set_default(2.0)
      .set_description("task status update interval to manager")
      .set_long_description("interval (in seconds) for sending mds task status to ceph manager"),
+
+    Option("mds_max_snaps_per_dir", Option::TYPE_INT, Option::LEVEL_ADVANCED)
```

Comment:
`TYPE_UINT`

Reply:
`diri->inode.rstat.rsnaps` is an `int64_t` and so I changed this back from `TYPE_UINT` to `TYPE_INT` since I got a warning for comparison between signed and unsigned numbers. Looks like I need to type-cast one of those for comparison.